src/msw/renderer.cpp

```diff
@@ -604,6 +604,11 @@ int wxRendererMSW::GetHeaderButtonHeight(wxWindow * WXUNUSED(win))
 
     wxON_BLOCK_EXIT1( ::DestroyWindow, hwndHeader );
 
+    // Must ensure the proper font is set or the wrong value will be returned
+    // At 200% scaling it was returning a height of 28 when it should have been 40
+    WXHANDLE hFont = (win && win->GetFont().IsOk() ? win->GetFont() : wxSystemSettings::GetFont(wxSYS_DEFAULT_GUI_FONT)).GetResourceHandle();
+    ::SendMessage(hwndHeader, WM_SETFONT, (WPARAM)hFont, MAKELPARAM(TRUE, 0));
```

Comment:
I probably should have looked up the usage. I just used the same parameters that were used elsewhere since I don't frequently call Win API functions directly. I doubt it really forces the control to redraw since the window is hidden, but it does make sense to not pass true.